src/modules/commander/Commander.cpp

```diff
@@ -2122,6 +2130,7 @@ Commander::run()
 		// engine failure detection
 		// TODO: move out of commander
 		orb_check(actuator_controls_sub, &updated);
+		actuator_controls_s actuator_controls = {};
```

Comment:
Why did the scope of `actuator_controls` change?

Reply:
Will reduce scope to minimum necessary.